packages/edit-site/src/components/sidebar-dataviews/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * WordPress dependencies
+ */
+import { __experimentalItemGroup as ItemGroup } from '@wordpress/components';
+import { page, columns } from '@wordpress/icons';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+
+/**
+ * Internal dependencies
+ */
+import { useLink } from '../routes/link';
+import { default as DEFAULT_VIEWS } from '../dataviews/default-views';
+import { unlock } from '../../lock-unlock';
+const { useLocation } = unlock( routerPrivateApis );
+import SidebarNavigationItem from '../sidebar-navigation-item';
+
+function DataViewItem( { dataview, isActive } ) {
+	const {
+		params: { path },
+	} = useLocation();
+	let icon;
```

Comment:
This seems like a good helper function here. We're going to need to get the icon per type in other places too. Example is the `view options`(top right) to change per type.